src/App.js

```diff
@@ -1,25 +1,23 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Routes, Route, Link } from 'react-router-dom';
+import AddItem from './components/AddItem';
+import ListView from './components/ListView';
+import Navigation from './components/Navigation';
+import Home from './components/Home';
+
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="addItem" element={<AddItem />} />
```

Comment:
Maybe we can circle back and look into updating route names here

Reply:
sure, we can discuss the names. If you have any ideas post it here or we discuss tomorrow during our meeting?